src-tauri/tests/tests/fixtures-load_dev_profile/states/load_dev_profile.json

```diff
@@ -0,0 +1,83 @@
+{
+    "active_profile": {
+        "name": "Ferris",
+        "picture": "&#129408",
+        "theme": "system",
+        "primary_did": "did:key:z6Mkg1XXGUqfkhAKU1kVd1Pmw6UEj1vxiLj1xc91MBz5owNY"
+      },
+      "locale": "en",
+      "credentials": [
+        {
+          "id": "33653133-6636-3633-6462-356231643533",
+          "issuer_name": null,
+          "format": {
+            "format": "jwt_vc_json"
+          },
+          "data": {
+            "@context": [
+              "https://www.w3.org/2018/credentials/v1",
+              "https://www.w3.org/2018/credentials/examples/v1"
+            ],
+            "type": [
+              "VerifiableCredential",
+              "DriverLicenseCredential"
+            ],
+            "issuer": "http://192.168.1.127:9090/",
+            "issuanceDate": "2022-08-15T09:30:00Z",
+            "expirationDate": "2027-08-15T23:59:59Z",
+            "credentialSubject": {
+              "id": "did:key:z6Mkg1XXGUqfkhAKU1kVd1Pmw6UEj1vxiLj1xc91MBz5owNY",
+              "licenseClass": "Class C",
+              "issuedBy": "California",
+              "validity": "Valid"
+            }
+          },
+          "metadata": {
+            "is_favorite": false,
+            "display": {
+              "icon": null,
+              "color": null,
+              "name": null
+            }
+          }
```

Comment:
Since the the values in metadata are all default values I think we should just omit them, same for metadata in line 69.

That means however that the `#[serde(default)]` attribute should be added above the`metadata` field in the `DisplayCredential` struct in `verifiable_credential_record.rs`.

Reply:
Same goes for `metadata` in `handle_oid4vp_1.json` and `handle_oid4vp_2.json`.